### ISSUE TITLE
Pom file version 1.5.31 has disappeared from public access.

### ISSUE THREAD

**Issue by kepocnhh** [Kotlin/dokka#2306]
Pom file version 1.5.31 has disappeared from public access.

**Expected behaviour**
Gradle downloads the pom file from the link https://plugins.gradle.org/m2/org/jetbrains/dokka/org.jetbrains.dokka.gradle.plugin/1.5.31/org.jetbrains.dokka.gradle.plugin-1.5.31.pom

As of 2022 January 12, there are no folders inside for versions 1.5.*

**Screenshots**
![photo_2022-01-12_19-08-18](https://user-images.githubusercontent.com/6218886/149177465-ee589fec-04b6-4a96-a7a2-0bec3919ba1f.jpg)

**To Reproduce**
Go to https://plugins.gradle.org/m2/org/jetbrains/dokka/org.jetbrains.dokka.gradle.plugin/1.5.31/org.jetbrains.dokka.gradle.plugin-1.5.31.pom and get 502 response code.

File build.gradle.kts:
```Kotlin
...
repositories {
    mavenCentral()
}

plugins {
    id("org.jetbrains.dokka") version "1.5.31"
}
...
```

**Additional context**
I have been using version 1.5.31 for a long time and there were no problems downloading this dependency. An example of ready-made documentation created using 1.5.31: https://stanleyprojects.github.io/AndroidExtension.UserInterface/documentation/0.0.9/index.html

**Comment by IgnatBeresnev** [comment]
Hi!

It seems like gradle plugin portal is experiencing partial outage (for repository and downloads), and previous versions of dokka might be affected. I think the newer versions are stored on healthy nodes, that's why they're available, but who knows. 

You can check the status here: https://status.gradle.com/

We'll keep an eye on this, thanks!

**Comment by vmishenev** [comment]
We have not published Dokka 1.4.32, 1.5.0, 1.5.30, 1.5.31 in Gradle Plugin portal (#1918).  These versions are available in Maven Central. Since Dokka 1.6.0 we have been publishing  in Gradle Plugin portal as well. See https://plugins.gradle.org/plugin/org.jetbrains.dokka

**Comment by kepocnhh** [comment]
Inside https://plugins.gradle.org/m2/org/jetbrains/dokka/org.jetbrains.dokka.gradle.plugin/ there are still no version 1.5.* folders.
But now if you try to explicitly go to https://plugins.gradle.org/m2/org/jetbrains/dokka/org.jetbrains.dokka.gradle.plugin/1.5.31 then you will be redirected to https://repo.gradle.org/ui/native/plugin-portal-backend/org/jetbrains/dokka/org.jetbrains.dokka.gradle.plugin/1.5.31 where there is a pom file. 🤷‍♂️

**Comment by IgnatBeresnev** [comment]
Yep, as Vadim has pointed out earlier, this is expected as versions < 1.6 were not published to gradle plugin portal directly.

The reason you were experiencing problems yesterday was because jcenter was down. This has been resolved and everything should work as expected now. 

> Resolved - JCenter is now back online, systems are fully operational.
> Jan 13, 06:35 UTC

Let us know if you have any more questions
